projects/fluxcd/flux2/Makefile

```diff
@@ -12,14 +16,26 @@ EXTRA_GO_LDFLAGS=-X main.VERSION=$(GIT_TAG:v%=%)
 FLUX_MANIFESTS_TARGET=$(REPO)/cmd/flux/manifests
 
 BINARY_PLATFORMS=linux/amd64 linux/arm64 darwin/amd64 darwin/arm64
+FETCH_BINARIES_TARGETS=eksd/kubernetes/client
 
 HAS_S3_ARTIFACTS=true
-IMAGE_NAMES=
+HAS_HELM_CHART=true
+HELM_DESTINATION_REPOSITORY=fluxcd/flux2
+HELM_SOURCE_OWNER=fluxcd-community
+HELM_SOURCE_REPOSITORY=helm-charts
+HELM_GIT_TAG?=$(shell cat HELM_GIT_TAG)
+HELM_DIRECTORY=charts/flux2
+HELM_ADDITIONAL_KEY_VALUES=\
+  FLUX_CLI_TAG=$(shell $(BASE_DIRECTORY)/build/lib/image_shasum.sh ${IMAGE_REPO} fluxcd/flux-cli latest) \
```

Comment:
Instead of hardcoding `latest` use `$(LATEST)`, this way its the current value depending on the branch.  On main its latest but on release branches its the release branch name.

Reply:
Done